pkg/kn/commands/service/create_test.go

```diff
@@ -1082,3 +1082,12 @@ func TestServiceCreateFromYAMLWithOverrideError(t *testing.T) {
 	assert.Assert(t, err != nil)
 	assert.Assert(t, util.ContainsAll(err.Error(), "expected", "0", "<=", "2147483647", "autoscaling.knative.dev/maxScale"))
 }
+
```

Comment:
What about create tests with bad (e.g., misspelled `@latst=100`) and other parsing issues?

Reply:
Thanks, I will add those